<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>word_register_window</class>
 <widget class="QWidget" name="word_register_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>489</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>注册账号</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#word_register_window{border-image: url(:/Yosemite-Color-Block.png);}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_account">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(30, 84, 145);/*设置边框的粗细，以及颜色*/
border-radius: 15px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
background-color: rgb(238, 192, 218);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(30, 84, 145);/*设置边框的粗细，以及颜色*/
border-radius: 15px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
background-color: rgb(238, 192, 218);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_repassword">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(30, 84, 145);/*设置边框的粗细，以及颜色*/
border-radius: 15px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
background-color: rgb(238, 192, 218);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_account">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 84, 145);</string>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 84, 145);</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_repassword">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 84, 145);</string>
   </property>
   <property name="text">
    <string>重复密码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_register">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 15pt &quot;微软雅黑&quot;;
    color: #1e5491;
    background-color: #eec0da;
    border-color: #567cbe;
    border-radius: 17px;
    border-style: solid;
    border-width: 3px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #1e5491;
    background-color: #c0a1cc;
    border-color: #567cbe;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #eddfed;
    background-color: #1c58a3;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_back">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 15pt &quot;微软雅黑&quot;;
    color: #1e5491;
    background-color: #eec0da;
    border-color: #567cbe;
    border-radius: 17px;
    border-style: solid;
    border-width: 3px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #1e5491;
    background-color: #c0a1cc;
    border-color: #567cbe;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #eddfed;
    background-color: #1c58a3;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
